Formula/vcflib.rb

```diff
@@ -2,32 +2,55 @@ class Vcflib < Formula
   desc "Command-line tools for manipulating VCF files"
   homepage "https://github.com/ekg/vcflib"
   url "https://github.com/ekg/vcflib.git",
-    tag: "v1.0.1", revision: "d150a89fa4f717634b06e1c78a37794d2c10c94c"
+    tag: "v1.0.2", revision: "da9929c00938b1d1b042e8ce42514211ef34875d"
 
   bottle do
     root_url "https://ghcr.io/v2/brewsci/bio"
     sha256 cellar: :any_skip_relocation, sierra:       "0a7dd127083d4e4515cbd3ed38bd5cc5bdc693e0f1fd1bb3266bf0e17e04922d"
     sha256 cellar: :any_skip_relocation, x86_64_linux: "f10f5737f8ecb6bf5ba530d10a86bb7576843c7e49f401c6018e660cf8027001"
   end
 
-  depends_on "gcc" if OS.mac?
+  depends_on "cmake" => :build
+
+  depends_on "htslib"
   depends_on "python"
   depends_on "xz"
 
   uses_from_macos "bzip2"
   uses_from_macos "perl"
   uses_from_macos "zlib"
 
-  fails_with :clang # error: ordered comparison between pointer and zero
+  patch :DATA
 
   def install
-    system "make"
-    pkgshare.install Dir["bin/*.R"]
-    pkgshare.install Dir["bin/*.r"]
-    rm Dir["bin/*.R"]
-    rm Dir["bin/*.r"]
-    bin.install Dir["bin/*"]
+    cd "tabixpp" do
```

Comment:
```
Errno::ENOENT: No such file or directory @ dir_chdir - tabixpp
```
https://github.com/brewsci/homebrew-bio/runs/4698088786?check_suite_focus=true#step:5:287

The directory `tabixpp` does not exist.
See https://github.com/vcflib/vcflib

Reply:
@sjackman Odd, as it seems to be building fine on macOS (both locally and in the CI). Just not Linux. Anyway, I see [this commit](https://github.com/ekg/tabixpp) which is relevant. How can I get the formula to check out tabixpp (say, the `v1.1.0` tag) in the root of the vcflib repo? Or should we create a separate formula for tabixpp?

Reply:
https://github.com/brewsci/homebrew-bio/runs/4698088830?check_suite_focus=true#step:4:285
```
  Error: An exception occurred within a child process:
285
    Errno::ENOENT: No such file or directory @ dir_chdir - tabixpp
286
  
287
==> brew audit brewsci/bio/vcflib --online --git --skip-style
288
Warning: install failed
289
==> SKIPPED brewsci/bio/vcflib
290
```
It seems to be failing on macOS as well in CI, though CI is reporting the run as successful for some strange unknown reason, possibly related to this `SKIPPED brewsci/bio/vcflib` message.

When testing locally, ensure that you've set `HOMEREW_NO_AUTO_UPDATE=1` to ensure that you're testing your changes. The auto update can `git stash` your changes when you run `brew install vcflib`.

Reply:
@sjackman Okay, thanks. Anyway, do you think we should create a separate formula for tabixpp? I can submit a PR for that too, if so.

Reply:
Yes, I think so. Is `tabixpp` a dependency of `vcflib`?

Reply:
It is.